x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_list.ts

```diff
@@ -23,27 +23,38 @@ interface SLOListParams {
   name?: string;
 }
 
-interface UseFetchSloListResponse {
+export interface UseFetchSloListResponse {
   loading: boolean;
   sloList: SLOList;
 }
 
-const useFetchSloList = (name?: string): UseFetchSloListResponse => {
+export function useFetchSloList({
+  name,
+  refetch,
+}: {
+  refetch: boolean;
+  name?: string;
+}): UseFetchSloListResponse {
+  const [sloList, setSloList] = useState(EMPTY_LIST);
   const params: SLOListParams = useMemo(() => ({ name }), [name]);
   const shouldExecuteApiCall = useCallback(
-    (apiCallParams: SLOListParams) => apiCallParams.name === params.name,
-    [params]
+    (apiCallParams: SLOListParams) => apiCallParams.name === params.name || refetch,
```

Comment:
Same as earlier. I believe this can be done cleaner but I want to do a POC with RQ first before I start optimizing this implementation